### ISSUE TITLE
Attached file gets deleted on screen rotation 

### ISSUE THREAD

**Issue by Olamatto** [itprojects/InboxPager#18]
#### Expected behavior
The attached file ought to stay saved on the application, on any screen orientation, be it portrait or landscape. 

#### Actual Behavior 
After attaching a file, on screen rotation the file gets missing immediately.

#### How to reproduce
- Download and install the application [InboxPager](https://f-droid.org/repository/browse/?fdfilter=inbox&fdid=net.inbox.pager) and [Ol file manager](https://play.google.com/store/apps/developer?id=OpenIntents)
- Lunch the application 
- Click on the option by the right hand corner 
- Select Add account
- Fill in all necessary details about the account
- Then click save
- Select the account
- Click on the option in the right hand corner 
- Select send message 
- Fill in all necessary details 
- Click on the Add attachment 
- Select the file using Ol file manager 
- After selection
- Rotate the screen to landscape 

And the attached file gets deleted immediately. 
<hr>

Application version:
InboxPager 2.2
Ol file manager 2.2.3

Operating system: Android v7.0

#### Recording Of The Bug
https://youtu.be/1UX0Od-FHA0

**Comment by itprojects** [comment]
Did you try to send the email?
Looking to confirm either:

- no attachment counting
- real no attachment sending

**Comment by itprojects** [comment]
This is a save/restore state of Activity bug due to the orientation being changed, but not handled by app. Fixing.

**Comment by Olamatto** [comment]
Yes it did not, ok good to hear that you fixing it,  i hope you fix it soon,  well done.

**Comment by itprojects** [comment]
Well, it should be fixed in version 3.0 (among other bugs), update coming soon.

**Comment by itprojects** [comment]
Fixed in v4.0, with the introduction of all-internal File Manager (Send/Save Attachments).
